codec-dns/src/main/java/io/netty/handler/codec/dns/DefaultDnsSrvRecord.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.dns;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.util.internal.StringUtil;
+
+/**
+ * Created by epenorr on 2016-08-06.
+ */
+public class DefaultDnsSrvRecord extends AbstractDnsRecord {
+
+    private final int priority;
+    private final int weight;
+    private final int port;
+    private final String target;
+
+    public DefaultDnsSrvRecord(String name, DnsRecordType type, int dnsClass, long timeToLive, ByteBuf in) {
```

Comment:
if users put it a map it would make sense.  That said other records not have it implemented so I think it is fine to also not do it here.